tests/functional/ext/code_style/cs_use_math_not_float.py

```diff
@@ -16,3 +16,8 @@
 upper_nan_float = float("NaN")  # [consider-math-not-float]
 typo_nan_float = float("nani")  # [consider-math-not-float]
 other_typo_nan_float = float("nna")  # [consider-math-not-float]
+
+
+# Regression test for https://github.com/pylint-dev/pylint/issues/10708
+x = slice(42)
+x()  # pylint: disable=not-callable
```

Comment:
```suggestion
```
If the other tests fail this one is going to behave the same so we don't need to test it twice.